CHANGELOG.md

```diff
@@ -8,6 +8,21 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.8] - 2024-08-07
```

Comment:
**Reminder:** You need to add the `[1.5.8]` link at the bottom of the CHANGELOG.

Reply:
This wasn't addressed 🤔

Reply:
@rhamzeh added it 🙇